qemu/tests/virtio_port_hotplug.py

```diff
@@ -0,0 +1,93 @@
+import time
+import logging
+
+from autotest.client.shared import error
+from avocado.core import exceptions
+
+from virttest.qemu_devices import qdevices
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+    Test hot unplug virtio serial devices.
+
+    1) Start guest with virtio serial device(s).
+    2) reload module in guest os.
+    3) Hot-unplug virtio serial port one by one.
+    4) unload module in guest
+    5) Hotplug it one by one.
+    6) reload module in guest.
+    7) Reboot VM to check guest kernel not panic.
```

Comment:
Seems the steps description is not correct, following is better:
1) Start guest with virtio serial device(s).
2) Load module in guest os.
3) For each of the virtio serial ports, do following steps one by one:
    3.1) Unload module in guest
    3.2) Hot-unplug the virtio serial port
    3.3) Hotplug the devices
    3.4) Reload module in the guest
4) Reboot VM to make sure the guest kernel not panic.